static/templates/sidebar/party-document-partial.hbs

```diff
@@ -1,6 +1,6 @@
 <ol class="party-list">
 {{#each parties}}
-    <li class="party flexcol {{#unless (lookup @root.extraFolders this.id)}}collapsed{{/unless}}" data-document-id="{{this.id}}">
+    <li class="party flexcol {{#unless (lookup @root.extraFolders this.id)}}collapsed{{/unless}}" data-entry-id="{{this.id}}" data-document-id="{{this.id}}">
```

Comment:
Here is the actual `this` crime!

Reply:
hah! Sure I'll try to replace it all. The thing was copied from the partials foundry has, and they use this! Perhaps its deviated enough to where I can replace the whole thing.